.env.example

```
@@ -18,6 +18,10 @@ NEXT_PUBLIC_SELECT_LANGUAGE="en"
 # Site URL (used for OGP and Twitter card image URLs)
 NEXT_PUBLIC_SITE_URL="https://aituberkit.com"
 
+# 公式サイト固有のSEOページを有効化（公式デプロイ以外ではfalse） /
+# Enable official-site-only SEO pages (keep false outside the official deployment)
+NEXT_PUBLIC_OFFICIAL_SITE="false"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)